roles/keystone/tasks/monitoring.yml

```diff
@@ -17,6 +17,19 @@
                args="--service keystone --criticality {{ keystone.monitoring.sensu_checks.check_keystone_api.criticality }}"
   notify: restart sensu-client
 
+- block:
+  - name: create directory for below symbolic link
+    file:
+      path: /opt/openstack/current/keystone/bin/
+      state: directory
+  
+  - name: add symolic link for keystone expired token check (rhel distro only)
+    file: 
+      src: /bin/python
+      dest: /opt/openstack/current/keystone/bin/python
```

Comment:
This sensu plugin is weird, it tried to extend keystone-manage command to query the count of expired token.  so it need keystone service related python env. And to install another keystone service in sensu venv is not a good idea.
Do you agree I use an new plugin to replace it, and the new plug will use mysql query directly?